components/html-block/test/html-block.vdiff.js

```diff
@@ -280,7 +280,8 @@ describe('d2l-html-block', () => {
 			html`<d2l-html-block style="width: 650px;" html="An equation...${mathInline} embedded inline with text, and showing placement of indicies for summations."></d2l-html-block>`,
 			{ mathjax: { renderLatex: true } }
 		);
-		await waitUntil(() => elem.clientHeight === 61);
+		console.log(elem.clientHeight, ', expected 61');
+		//await waitUntil(() => elem.clientHeight === 61);
```

Comment:
You may want to put the console inside the `waitUntil` -- that way you see the different values it's getting over time.

Reply:
Yeah I wasn't expecting them to change after being non-zero, but I'll make some changes to see what the final result is.